Postfix/postfix/tests/filename3.json

```diff
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "message": "707A12000A: warning: header Content-Disposition: attachment;??filename=\"?iso-8859-2?q?representative_on_migration.pdf?=\"; size=259210;?? from local; from=<photo@mordor.com> to=<Pipin.touque@lacomte.net>"
+  },
+  "expected": {
+    "message": "707A12000A: warning: header Content-Disposition: attachment;??filename=\"?iso-8859-2?q?representative_on_migration.pdf?=\"; size=259210;?? from local; from=<photo@mordor.com> to=<Pipin.touque@lacomte.net>",
+    "event": {
+      "category": [
+        "email"
+      ],
+      "kind": "event",
+      "type": [
+        "info"
+      ]
+    },
+    "email": {
+      "from": {
+        "address": [
+          "photo@mordor.com"
+        ]
+      },
+      "to": {
+        "address": [
+          "Pipin.touque@lacomte.net"
+        ]
+      }
+    },
+    "file": {
+      "name": "?iso-8859-2?q?representative_on_migration.pdf?=",
```

Comment:
I think the `file.name` should end with `.pdf`

Reply:
As hideous as it is it, this value was provided in a sample and I was able to create a file with this name. Not sure if we have to truncate this.